Main_Tasks/Main_Data_Classes/main_data_classes_task.rb

```diff
@@ -54,15 +54,14 @@ def even_odd_by_index(array)
   print "Even: #{even}\nOdd: #{odd}"
 end
 
-#returns index of last elem which follows the condition a[0] < elem < a[-1]
-def last_element_in_range_index(array)
-  range = (array.first+1...array.last)
-  indexes_of_elements_in_range = []
+#returns index of last elem of a given array which follows the condition a[0] < elem < a[-1]
+def last_element_conditional_index(array)
```

Comment:
@d-konovalov changed the condition, now checking the exact occurrence without using range